src/api/java/com/minecolonies/api/colony/managers/interfaces/IEventDescriptionManager.java

```diff
@@ -0,0 +1,42 @@
+package com.minecolonies.api.colony.managers.interfaces;
+
+import com.minecolonies.api.colony.colonyEvents.descriptions.IColonyEventDescription;
+
+import net.minecraft.nbt.CompoundNBT;
+import org.jetbrains.annotations.NotNull;
+
+import java.util.List;
+
+/**
+ * Interface for the event description manager, the event description manager deals the colony event log events.
+ */
+public interface IEventDescriptionManager
```

Comment:
Again missing the `INBTSerializable<CompoundNBT>` instead of using the custom methods.

Reply:
done.